.chloggen/3218-align-label-filter-flag.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Depreceate label flag and align label filter flag with attribute filter flag name"
```

Comment:
please explicitly mention the deprecated flag and the new flag.

Reply:
mention when the deprecated flag will be removed and create a ticket for it.

Reply:
@pavolloffay  Thank you for your review!
Do you know when the deprecated flag will be removed? Is there an exact version or something like that?
Or should I just say it will be removed when the new ticket #XXXXX is resolved?

Reply:
Explicitly mentioning the flags
Created new issue #3236 and also mentioning that in the note

Reply:
@pavolloffay @jaronoff97 
Let me know if I still need to modify something